.env

```
@@ -8,10 +8,10 @@ APP_DESCRIPTION=Visualization, Exploration, and Data Analysis (VEDA)
 APP_CONTACT_EMAIL=email@example.org
 
 # Endpoint for the Tiler server. No trailing slash.
-API_RASTER_ENDPOINT='https://staging-raster.delta-backend.com'
+API_RASTER_ENDPOINT='https://openveda.cloud/api/raster'
```

```suggestion
API_RASTER_ENDPOINT='https://staging.openveda.cloud/api/raster'
```
New url to preview veda staging catalog